source/best-practices/backup.rst

```diff
@@ -111,6 +112,24 @@ Windows
 robocopy
 ^^^^^^^^
 
+使用 Windows 自带的 `robocopy <https://docs.microsoft.com/zh-cn/windows-server/administration/windows-commands/robocopy>`__
+命令进行增量备份。这意味着只有第一次备份的时候需要花比较多的时间来同步文件，之后再使用该命令进行备份
+只会同步有改动的文件。假设要备份整个 D 盘，移动硬盘下的备份目录为 :file:`F:\\backup` 。
+打开 CMD 或 PowerShell，使用以下命令可以将 D 盘同步到备份目录下，此时备份目录是 D 盘的一个镜像::
+
+    $ robocopy D:\ F:\backup /mir /mt /A-:H /XD Config.Msi $RECYCLE.BIN WSLDIR
```

Comment:
I suggest removing the `WSLDIR`, because it's created by users.
```suggestion
    $ robocopy D:\ F:\backup /mir /mt /A-:H /XD Config.Msi $RECYCLE.BIN
```

Reply:
@zhaozhiyuan1989，`robocopy` cannot copy WSL, right?

Reply:
The answer may be yes, but the directory `WSLDIR` is created by users. I think the default WSL dir is in `C:`, no?

Reply:
> The answer may be yes, but the directory `WSLDIR` is created by users. I think the default WSL dir is in `C:`, no?

Yes, I'd like to put it in the note if the answer is yes.

Reply:
Revised in e82e552.

Reply:
robocopy 可以备份 WSL2，但是由于 WSL2 是一个整体的 ext4.vhdx 文件，所以只要稍微改动就会备份整个系统，所以我不建议备份它